plugins/test/unit/plugins/importers/yum/test_sync.py

```diff
@@ -1,7 +1,11 @@
 from copy import deepcopy
 from cStringIO import StringIO
 import os
-import unittest
+
+try:
+    import unittest2 as unittest
+except ImportError:
```

Comment:
The deal with this is that you don't need ``unittest`` if you have python 2.7. ``unittest2`` is just a backport of 2.7's ``unittest`` module.

You can usually install ``unittest2`` with python2.7 for compatibility, so we could just require that and skip this import complication. The downside there is that it's a requirement we'd express in our spec file and similar, whose only value would be to simplify these imports.

I could go either way.